manifests/params.pp

```diff
@@ -11,6 +11,11 @@
         $service_file_template = 'confluence/confluence.service.erb'
         $service_lockfile      = '/var/lock/subsys/confluence'
         $refresh_systemd       = true
+      } elsif $facts['os']['release']['major'] == '2' {
```

Comment:
is there another attribute that we can check too? This does meet the standard that the other version checkers are doing but it seems so vague to look for '2' vs '6' especially when 2 is newer than 6